test/common/router/config_impl_test.cc

```diff
@@ -55,6 +55,26 @@ TEST(RouteMatcherTest, TestRoutes) {
       ]
     },
     {
+      "name": "wildcard",
+      "domains": ["*.foo.com", "*-bar.baz.com"],
+      "routes": [
```

Comment:
Same as prev question. From cursory look, it feels intuitive to me to view wildcard as a empty or non empty set. But it's just me (one data point).